fs/fs.go

```diff
@@ -115,7 +118,15 @@ func NewFsInfo(context Context) (FsInfo, error) {
 	fsInfo.addDockerImagesLabel(context, mounts)
 
 	glog.Infof("Filesystem partitions: %+v", fsInfo.partitions)
-	fsInfo.addSystemRootLabel(mounts)
+	rootDevice, err := fsInfo.GetDirFsDevice(context.RootPath)
+	if err != nil {
+		return nil, fmt.Errorf("Error trying to get filesystem Device for rootPath %v: err: %v", context.RootPath, err)
```

Comment:
nit: error messages should start with a lowercase character (log messages start with Capital)

Same below.